dicom_utils/anonymize.py

```diff
@@ -45,12 +46,21 @@ def age_to_anonymized_age(age_str: str) -> str:
 
 
 TagTuple = Tuple[int, int]
```

Comment:
Just spotted this... could you remove this definition of TagTuple since it's not needed anymore?